packages/account-v2/src/components/SignupWizard/index.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { useEffect } from 'react';
+import ReactModal from 'react-modal';
+import { StandaloneXmarkBoldIcon } from '@deriv/quill-icons';
+import { useSignupWizardContext } from '../../context/SignupWizardContext';
+import Actions from './Actions';
+import { CUSTOM_STYLES } from './helpers';
+import './index.scss';
+
+const SignupWizard: React.FC = () => {
+    const {
+        currentStep,
+        helpers: { goToNextStep },
+        setIsWizardOpen,
+    } = useSignupWizardContext();
+    useEffect(() => {
+        ReactModal.setAppElement('#v2_modal_root');
+    }, []);
+    return (
+        <ReactModal
+            isOpen
```

Comment:
should'nt we bing this is_open with `isWizardOpen` from context?

Reply:
Updated!